modules/authentication/src/handlers/oauth2/interfaces/OAuth2Settings.ts

```diff
@@ -44,6 +47,9 @@ export class OAuth2Settings {
     this.responseType = providerParams.responseType;
     this.responseMode =
       providerParams.responseMode === 'form_post' ? 'form_post' : 'query';
+    this.privateKey = providerConfig.privateKey;
```

Comment:
override OAuth2Settings and create a new class for apple. These params make no sense for other providers